sites/default/menu_data.json

```diff
@@ -830,7 +830,7 @@
         "label" : "Portal Dashboard",
```

Comment:
So, this link is broken, but I couldn't figure out what it should be. /provider is not right, because it doesn't exist. I looked back through history of file, and the earliest it *appeared* was in this commit https://github.com/LibreHealthIO/LibreEHR/commit/fbc531ed87d8e73f15422d8058c457a3ed206a75#diff-909487e913474f47f5e1aa1d3e73e0a4 from what I see.

@teryhill Can you let me know what I need to do here?

Reply:
Yeah @teryhill 
Where is this portal dashboard thingie?

> Internal Server Error
> 
> The server encountered an internal error or misconfiguration and was unable to complete your request.
> 
> Please contact the server administrator at webmaster@suncoastconnection.com to inform them of the time this error occurred, and the actions you performed just before this error.
> 
> More information about this error may be available in the server error log.

Reply:
under the miscellaneous drop down.